src/Section.js

```diff
@@ -38,7 +38,12 @@ class Section extends React.Component {
                     onClickNewCard={this.onClickNewCard} 
                     onClickSort={this.onClickSort}                        
                 />
-                {(createPanel === status.name) && <CreatePanel status={status.name} />}
+                {(createPanel === status.name) && 
```

Comment:
why do you have parentheses around first expression and not around second? you should have for the second only